### PR TITLE
Removing multi-thread flags in PETSC and HYPRE 5341

### PR DESCRIPTION
**Pull request by mbairdphys**
Damien, please have a look.  This should correct the CTF timeouts on LEZA without add an env variable

### CODE REVIEW

**Comment by dalg24** [comment]
We are still passing a flag to configure HYPRE with OpenMP:

https://github.com/CASL/vera_tpls/blob/feb2a9e4cf826a74a28f0487e0bf44d9a46bb4a6/TPL_build/TPLs/HYPRE.cmake#L35-L41

I will remove L41 and give it a try.

**Comment by dalg24** [comment]
Also what's the deal with that `CONFIGURE_HYPRE` variable?

https://github.com/CASL/vera_tpls/blob/feb2a9e4cf826a74a28f0487e0bf44d9a46bb4a6/TPL_build/CMakeLists.txt#L64-L66

I just realized we do not build HYPRE, we let PETSc download it and install it as an external package...

**Comment by mbairdphys** [comment]
Hi Damien, 

  Correct, PETSC downloads HYPRE, along with MUMPS, superlu-dist, etc...  Someone could potentially build hyper with petsc, so I made them consistent.  (Not vary likely at all I agree)

File: TPL_build/TPLs/HYPRE.cmake [hunk 1]
```diff
@@ -61,9 +60,9 @@ IF ( CMAKE_BUILD_HYPRE )
         LIST(APPEND CONFIGURE_OPTIONS )
     ENDIF()
     LIST(APPEND CONFIGURE_OPTIONS CC=mpicc CXX=mpicxx FC=mpif90 F77=mpif77 F90=mpif90 )
-    LIST(APPEND CONFIGURE_OPTIONS "CFLAGS=-fPIC -fopenmp" )
-    LIST(APPEND CONFIGURE_OPTIONS "CXXFLAGS=-fPIC -fopenmp" )
-    LIST(APPEND CONFIGURE_OPTIONS "FFLAGS=-fopenmp" )
+    LIST(APPEND CONFIGURE_OPTIONS "CFLAGS=-fPIC" )
+    LIST(APPEND CONFIGURE_OPTIONS "CXXFLAGS=-fPIC" )
+    LIST(APPEND CONFIGURE_OPTIONS "FFLAGS=-fPIC" )
```
Comment (dalg24) [inline TPL_build/TPLs/HYPRE.cmake]: I assume `-fPIC` change for the Fortran flags was intended.  Anyway, HYPRE seems to be mostly a C/C++ project.

**Pull request merged**